pos_ticket_without_price/static/src/js/screens.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2021 Initos Gmbh
+   License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+*/
+
+odoo.define("pos_ticket_without_price.screens", function (require) {
```

Comment:
```suggestion
odoo.define("pos_ticket_without_price.ReceiptScreen", function (require) {
```

Please also rename the file and the assets entry